brains/shell.yml

```diff
@@ -44,21 +44,21 @@
 
   - name: "start-music"
     signals:
-      - order: "mais nous de la musique"
+      - order: "mets nous de la musique"
```

Comment:
Here, this is not a spelling mistake. It's due to the fact the the STT engine doen't know we want to use a verb. See https://github.com/kalliope-project/kalliope/blob/master/Docs/signals.md